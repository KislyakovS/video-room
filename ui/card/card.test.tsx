import { render } from '@testing-library/react';

import Card from './card';

describe('Component Card:', () => {
    it('snapshot', () => {
        const { container } = render(<Card
            poster='http://poster.com'
            title='Зеленая миля'
            duration='189 мин.'
        />);

        expect(container).toMatchSnapshot();
    })
})