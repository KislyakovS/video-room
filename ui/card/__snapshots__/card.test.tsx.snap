// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card> snapshot 1`] = `
<div>
  <article
    class="wrapper"
  >
    <div
      class="poster"
    >
      <img
        alt="Movie poster - Зеленая миля"
        decoding="async"
        loading="lazy"
        src="http://poster.com"
      />
      <span
        class="duration"
      >
        189 мин.
      </span>
    </div>
    <h3
      class="title"
      title="Зеленая миля"
    >
      Зеленая миля
    </h3>
  </article>
</div>
`;
